presto-native-execution/presto_cpp/main/types/tests/CMakeLists.txt

```diff
@@ -129,5 +129,6 @@ target_link_libraries(
   presto_types
   velox_dwio_common
   velox_exec_test_lib
+  velox_functions_prestosql
```

Comment:
Why do we need this dependency ? Seems a bit odd.

Reply:
This might be a mistake.  A similar dependency was necessary in an internal build for the function register* calls in the test, but those were pre-existing so probably not needed here.  I'll try removing it.